lib/asar.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-const fs = require('fs')
+const fs = process.versions.electron ? require('original-fs') : require('fs')
```

Comment:
I don't believe it would be worth it. The condition is still pretty short & straightforward and can be inlined.